tests/mssqltestutils.py

```diff
@@ -78,12 +78,30 @@ def create_test_db():
         '-', '_').replace('.', '_')
     test_db_name = u'mssqlcli_testdb_{0}_{1}'.format(
         local_machine_name, random_str())
-    query = u"CREATE DATABASE {0};".format(test_db_name)
-    for _, _, _, _, is_error in client.execute_query(query):
-        if is_error is True:
-            test_db_name = None
+    query_db_create = u"CREATE DATABASE {0};".format(test_db_name)
+    rows, columns, status, statement, is_error = None, None, None, None, None
+    count = 0
+
+    # retry logic in case db create fails
+    while count < 5:
+        for _, _, _, _, is_create_error in client.execute_query(query_db_create):
+            if not is_create_error:
```

Comment:
how about this... if it is a create error, can we log the error message, so we know what's gone wrong?

Reply:
Like this. I'll update.